packages/wrangler/src/deployment-bundle/worker.ts

```diff
@@ -4,7 +4,7 @@ import type { Json } from "miniflare";
 /**
  * The type of Worker
  */
-export type CfScriptFormat = "modules" | "service-worker";
+export type CfScriptFormat = "modules" | "service-worker" | "python";
```

Comment:
I'm not sure this is quite right—a python worker should still be a modules worker, I think?

Reply:
I guess it really depends what the meaning for this type is. I think differentiating `module` (JS) vs. `modules` (Python) makes sense since the code which reads these ultimately does a completely different thing depending on its value. Maybe renaming "python" to "modules-python" would be best?